### PR TITLE
MONA 138

### PR DESCRIPTION
**Pull request by mstone6769**

### CODE REVIEW

File: lib/src/iui-wizard/iui-wizard.html [hunk 1]
```diff
@@ -91,17 +49,14 @@ <h2 class="wizard-current-step wizard-desktop-hide">
...
-        class="btn btn-simple btn-wizard-options-menu">
+        ng-class="{'btn-active': wizard.dropdown.isOpen}">
         <svg class="icon">
-          <use
-            xlink:href="#icon-arrow-top"
-            ng-if="wizard.dropdown.isOpen" />
-          <use
-            xlink:href="#icon-arrow-bottom"
-            ng-if="!wizard.dropdown.isOpen" />
+          <use xlink:href="#icon-arrow-bottom" />
```
Comment (mcastre) [inline lib/src/iui-wizard/iui-wizard.html]: good

File: lib/src/wizard-service/wizard-service.test.js [hunk 2]
```diff
@@ -0,0 +1,294 @@
...
+        $provide.constant('wizardSettings', wizardSettings);
+      });
+      inject(function(_wizardService_, _$rootScope_) {
+        wizardService = _wizardService_;
+        $rootScope = _$rootScope_;
+      });
+    });
+
+    describe('when calling getStepClass(step, currentMajorStep, currentStep)', function() {
+      describe('where step={completed: true}; currentMajorStep = step; currentStep = step', function() {
```
Comment (mcastre) [inline lib/src/wizard-service/wizard-service.test.js]: these test suite names are pretty hard to follow. Can we rewrite them to describe what the following tests will be?

Comment (mstone6769) [inline lib/src/wizard-service/wizard-service.test.js]: can you give me an example of how to write it?

Comment (mcastre) [inline lib/src/wizard-service/wizard-service.test.js]: instead of throwing code in the name, maybe something like:
`where wizard step is complete`

File: lib/src/wizard-service/wizard-service.test.js [hunk 3]
```diff
@@ -0,0 +1,294 @@
...
+        });
+        it('should contain the active className', function() {
+          function filterClassName(className) {
+            return className === 'item-active';
+          }
+          expect(result.filter(filterClassName).length).toBe(1);
+        });
+      });
+
+      describe('where step={completed: false}; currentMajorStep = step; currentStep = step', function() {
```
Comment (mcastre) [inline lib/src/wizard-service/wizard-service.test.js]: This could be: `'where wizard step is not complete'`

Comment (mstone6769) [inline lib/src/wizard-service/wizard-service.test.js]: so `where wizard step is not complete and the current major step and current step is the same as the wizard step`
